models/error.go

```diff
@@ -280,3 +283,15 @@ func ErrFetchData(err error) error {
 func ErrSessionCopy(err error) error {
 	return errors.New(ErrSessionCopyCode, errors.Alert, []string{"Error: session copy error"}, []string{err.Error()}, []string{}, []string{})
 }
+
+func ErrSeedingPatterns(err error) error {
+	return errors.New(ErrSeedingPatternsCode, errors.Alert, []string{"Error while seeding patterns from seed content"}, []string{err.Error()}, []string{}, []string{})
+}
+
+func ErrGettingSeededPatterns(err error) error {
+	return errors.New(ErrGettingSeededPatternsCode, errors.Alert, []string{"Error while reading seeded patterns."}, []string{err.Error()}, []string{}, []string{})
```

Comment:
Can we add probable cause and suggested remediations?

Reply:
sure

Reply:
Probable cause: 

"Lack of Internet connectivity to download sample patterns, filters, and applications."

Reply:
Probably cause: "Invalid sample content."